tests/unit/amo/reducers/test_home.js

```diff
@@ -0,0 +1,70 @@
+import reducer, {
```

Comment:
I think labelling this `homeReducer` will improve the test's readability a lot. We do that elsewhere, I think it's worth doing here too.